.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -115,119 +115,66 @@ permissions:
+      commit-id: ${{ github.sha }}
+      commit-id-short: ${{ steps.commit-id.outputs.short }}
     steps:
       - name: Harden Runner
         uses: step-security/harden-runner@f086349bfa2bd1361f7909c78558e816508cdc10 # v2.8.0
         with:
           egress-policy: audit
 
-      - name: Install Semantic Version Tools
-        run: |
-          echo "::group::Download SemVer Binary"
-          sudo curl -L -o /usr/local/bin/semver https://raw.githubusercontent.com/fsaintjacques/semver-tool/master/src/semver
-          echo "::endgroup::"
-          echo "::group::Change SemVer Binary Permissions"
-          sudo chmod -v +x /usr/local/bin/semver
-          echo "::endgroup::"
-          echo "::group::Show SemVer Binary Version Info"
-          semver --version
-          echo "::endgroup::"
-
-      - name: Verify Workflow Parameters
-        id: parameters
-        run: |
-          BRANCH_NAME="${{ github.head_ref || github.ref_name }}"
-          BRANCH_NAME="${BRANCH_NAME##origin/}"
-          BRANCH_NAME_LOWER="$(echo "${BRANCH_NAME}" | tr '[:upper:]' '[:lower:]')"
-          BRANCH_NAME_SAFE="$(echo "${BRANCH_NAME_LOWER}" | tr '/' '-' | tr '_' '.')"
-
-          COMMIT_PREFIX="adhoc"
-          [[ "${{ inputs.version-policy }}" == "branch-commit" ]] && COMMIT_PREFIX="${BRANCH_NAME_SAFE}"
-
-          echo "branch-name=${BRANCH_NAME}" >>"${GITHUB_OUTPUT}"
-          echo "branch-name-lower=${BRANCH_NAME_LOWER}" >>"${GITHUB_OUTPUT}"
-          echo "branch-name-safe=${BRANCH_NAME_SAFE}" >>"${GITHUB_OUTPUT}"
-          echo "commit-prefix=${COMMIT_PREFIX}" >>"${GITHUB_OUTPUT}"
-          echo "commit-id=${{ github.sha }}" >>"${GITHUB_OUTPUT}"
-          echo "commit-id-short=$(echo "${{ github.sha }}" | cut -c1-8)" >>"${GITHUB_OUTPUT}"
-
-      - name: Verify Version Update (As Specified)
-        if: ${{ inputs.version-policy == 'specified' && !cancelled() && !failure() }}
-        run: |
-          VALID_VERSION="$(semver validate "${{ inputs.new-version }}")"
-
-          if [[ "${VALID_VERSION}" != "valid" ]]; then
-            echo "::error title=Version Error::The supplied new-version parameter (${{ inputs.new-version }}) is invalid and does not conform to the semantic versioning specifications."
-            exit 2
-          fi
-
       - name: Checkout Code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-
-      - name: Setup Java
-        uses: actions/setup-java@8df1039502a15bceb9433410b1a100fbe190c53b # v4.5.0
         with:
-          distribution: ${{ inputs.java-distribution }}
-          java-version: ${{ inputs.java-version }}
+          fetch-depth: 0
 
-      - name: Setup Gradle
-        uses: gradle/actions/setup-gradle@dbbdc275be76ac10734476cc723d82dfe7ec6eda # v3.4.2
+      - name: Setup Node.js
+        uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
         with:
-          gradle-version: ${{ inputs.gradle-version }}
```

Comment:
Still need the setup java and setup gradle steps for the branch commit checks.

Reply:
I've revamped `node-zxc-build-release-artifact.yaml` and these steps are kept as they were.